moveit_core/trajectory_processing/include/moveit/trajectory_processing/ruckig_traj_smoothing.h

```diff
@@ -64,13 +87,21 @@ class RuckigSmoothing
 
   /**
    * \brief Initialize Ruckig position/vel/accel. This initializes ruckig_input and ruckig_output to the same values
-   * \param rucking_input   Input parameters to Ruckig. Initialized here.
-   * \param ruckig_output   Output from the Ruckig algorithm. Initialized here.
    * \param first_waypoint  The Ruckig input/output parameters are initialized to the values at this waypoint
    * \param joint_group     The MoveIt JointModelGroup of interest
+   * \param[out] rucking_input   Input parameters to Ruckig. Initialized here.
+   * \param[out] ruckig_output   Output from the Ruckig algorithm. Initialized here.
+   */
+  static void initializeRuckigState(const moveit::core::RobotState& first_waypoint,
+                                    const moveit::core::JointModelGroup* joint_group,
+                                    ruckig::InputParameter<0>& ruckig_input, ruckig::OutputParameter<0>& ruckig_output);
```

Comment:
Is there a reason to use `0` rather than `ruckig::DynamicDOFs`? I would unify them for consistency

Reply:
Good point. I did a search and updated that everywhere